tests/scripts/test-e2e-cluster.sh

```diff
@@ -28,8 +28,9 @@ function run_suites() {
   run_suite "azure_openai" "" "azure_openai" "$AZUREOPENAI_PROVIDER_KEY_PATH" "https://ols-test.openai.azure.com/" "" "0301-dep" "gpt-3.5-turbo" "$OLS_IMAGE"
   (( rc = rc || $? ))
 
-  run_suite "bam" "" "bam" "$BAM_PROVIDER_KEY_PATH" "" "" "" "ibm/granite-13b-chat-v2" "$OLS_IMAGE"
-  (( rc = rc || $? ))
+  # BAM tests disabled temporarily
```

Comment:
when doing this in the future, please create a jira bug to re-enable or follow up on this, and reference it in the code you're commenting out.

we don't want to lose track of the fact that we disabled this, or why we disabled it and under what conditions we will re-enable it.